.github/workflows/publish_package.yml

```diff
@@ -1,31 +1,34 @@
-name: Publish cookietemple to PyPI
+name: Publish Cookietemple to PyPI
```

Comment:
Why uppercase? Think we settled on lowercase absolutely everywhere for consistency